lib/src/test/java/com/tbruyelle/rxpermissions2/RxPermissionsTest.java

```diff
@@ -20,8 +20,6 @@
 import android.content.pm.PackageManager;
 import android.os.Build;
 
-import com.tbruyelle.rxpermissions.BuildConfig;
```

Comment:
@wangyongf `BuildConfig` is being used in the line `50` how is this resolved? should `com.tbruyelle.rxpermissions2.BuildConfig` be imported?

Reply:
nvm looks like all the tests passes on travis